Unable to compile using build.cmd file

.NET Core Version: 3.1.200-preview-015002
Windows version: 10.0.18363
Does the bug reproduce also in WPF for .NET Framework 4.8?: Yes/No
Is this bug related specifically to tooling in Visual Studio (e.g. XAML Designer, Code editing, etc...)? No

Compiling using build.cmd, I'm getting the error: **"The .NET Core SDK cannot be found. Verify that it is installed and that any version specified by global.json matches the installed version"**

Trying to compile by Visual Studio the error is:
**"error NETSDK1045: The current .NET SDK does not support targeting .NET Core 5.0.  Either target .NET Core 3.1 or lower, or use a version of the .NET SDK that supports .NET Core 5.0."**

It seems that **Net Core 5** version is required, but no version is available for download.

Thank you.

Compilping by Visual Studio (16.5.0 Preview 5, but it is the same with 16.4.5) the error is: **C:\Program Files\dotnet\sdk\3.1.200-preview-015002\Sdks\Microsoft.Net.Sdk\targets\Microsoft.NET.TargetFrameworkInference.targets(127,5): error NETSDK1045: The current .NET SDK does not support targeting .NET Core 5.0.  Either target .NET Core 3.1 or lower, or use a version of the .NET SDK that supports .NET Core 5.0.**

Using build.cmd the first error is: **D:\Sources\NetFoundation\dotnet\wpf\src\Microsoft.DotNet.Wpf\src\System.Xaml\System.Xaml.csproj : error : The .NET Core SDK cannot be found. Verify that it is installed and that any version specified by global.json matches the installed version.
D:\Sources\NetFoundation\dotnet\wpf\src\Microsoft.DotNet.Wpf\src\System.Xaml\System.Xaml.csproj : error MSB4236: The specify SDK 'Microsoft.NET.SDK' has not been found.**

--- a/global.json
+++ b/global.json
@@ -11,6 +11,9 @@
       "version": "16.1"
     }
   },
+  "sdk": {
+    "version": "5.0.100-alpha1-015515"
+  },
   "msbuild-sdks": {
     "Microsoft.DotNet.Arcade.Sdk": "5.0.0-beta.20154.1",
     "Microsoft.DotNet.Helix.Sdk": "5.0.0-beta.20154.1"

